enums.ts

```diff
@@ -1,12 +1,12 @@
 export enum NodeTypes {
   callout = 'rdme-callout',
   codeTabs = 'code-tabs',
-  embed = 'embed',
+  embedBlock = 'embed-block',
```

Comment:
Is there a regular embed block still?

Reply:
ahh, no. i changed the structure enough that the serializing/deserializing was getting to be spaghetti with the current/old `embed` block, so i just moved it all wholesale to a new block/block type